src/snapred/backend/recipe/algorithm/GroupDiffractionCalibration.py

```diff
@@ -170,6 +183,9 @@ def PyExec(self) -> None:
         - PreviousCalibrationTable: str -- the name of the table workspace with previous DIFC values
         output:
         - OutputWorkspace: str -- the name of the diffraction-focussed d-spacing data after the final calibration
+        - MaskWorkspace: str -- the name of the mask workspace for detectors failing calibration (1.0 => dead-pixel, 0.0 => live-pixel)
+            when the mask workspace already exists, incoming masked values will be combined with any new masked values detected during
+            execution
```

Comment:
I believe indentations in this block are causing the docs build to fail

Reply:
I'll try that and see if it solves the issue.